Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
> it checks if the OS is 64 bit...

I see... I misread what you posted.

I guess that should work on all platforms then right?